docs/topics/proc_deploying-the-crud-booster-using-the-oc-cli-client.adoc

```diff
@@ -6,11 +6,12 @@
 
 
 [id='deploying-the-crud-booster-using-the-oc-cli-client_{context}']
-= Deploying the {name-mission-crud} booster using the `oc` CLI client
+[id='deploying-the-crud-example-using-the-oc-client_{context}']
+= Deploying the {name-mission-crud} example using the `oc` CLI client
```

Comment:
Deploying the {name-mission-crud} example--> Deploying the {name-mission-crud} example application